R/Utilities.R

```diff
@@ -374,13 +379,25 @@ ConvertSpecialChars <- function(.data,col){
         (grepl("[A-Za-z]", masked) == TRUE) ~ as.character("Text"),
         (grepl("%", masked) == TRUE) ~ as.character("Percentage"),
         (grepl(",", masked) == TRUE) ~ as.character("Comma-Separated Numeric"),
+        (grepl('\\d\\-\\d',masked) == TRUE) ~ as.character("Numeric Range - Averaged"),
+        (!stringi::stri_enc_mark(masked)%in%c("ASCII")) ~ as.character("Non-ASCII Character(s)"),
         TRUE ~ "Coerced to NA"
       ))
     
+    # Deal with numeric ranges
```

Comment:
Can you elaborate /add to the annotation here. How are we now dealing with/using numeric ranges?

Reply:
Added more details in this comment!